examples/seldon_deployment/pipeline.py

```diff
@@ -0,0 +1,452 @@
+    """Trainer params"""
+
+    solver: str = "saga"
+    penalty: str = "l1"
+    C: float = 1.0
+    tol: float = 0.1
+
+
+@step
+def sklearn_trainer(
+    config: SklearnTrainerConfig,
+    x_train: np.ndarray,
+    y_train: np.ndarray,
+) -> ClassifierMixin:
+    """Train SVC from sklearn."""
+    clf = LogisticRegression(
+        penalty=config.penalty, solver=config.solver, tol=config.tol, C=config.C
+    )
+    clf.fit(x_train.reshape((x_train.shape[0], -1)), y_train)
+    return clf
+
+
+@step
+def sklearn_evaluator(
+    x_test: np.ndarray,
+    y_test: np.ndarray,
+    model: ClassifierMixin,
+) -> float:
+    """Calculate accuracy score with classifier."""
+
+    test_acc = model.score(x_test.reshape((x_test.shape[0], -1)), y_test)
+    return test_acc
+
+
+class DeploymentTriggerConfig(BaseStepConfig):
+    """Parameters that are used to trigger the deployment"""
+
+    min_accuracy: float
+
+
+@step
+def deployment_trigger(
+    accuracy: float,
+    config: DeploymentTriggerConfig,
+) -> bool:
+    """Implements a simple model deployment trigger that looks at the
+    input model accuracy and decides if it is good enough to deploy"""
+
+    return accuracy > config.min_accuracy
+
+
+class SeldonDeployerConfig(BaseStepConfig):
+    """Seldon model deployer step configuration
+
+    Attributes:
+        model_name: the name of the Seldon model logged in the Seldon artifact
+            store for the current pipeline.
+        replicas: number of server replicas to use for the prediction service
+        implementation: the Seldon Core implementation to use for the prediction
+            service
+    """
+
+    model_name: str = "model"
+    replicas: int = 1
+    implementation: str
+    secret_name: Optional[str]
+    kubernetes_context: Optional[str]
+    namespace: Optional[str]
+    base_url: str
+    timeout: int
+    step_name: Optional[str]
+
+
+@step(enable_cache=True)
+def seldon_model_deployer(
+    deploy_decision: bool,
+    config: SeldonDeployerConfig,
+    model: ModelArtifact,
+    context: StepContext,
+) -> SeldonDeploymentService:
+    """Seldon Core model deployer pipeline step
+
+    Args:
+        deploy_decision: whether to deploy the model or not
+        config: configuration for the deployer step
+        model: the model artifact to deploy
+        context: pipeline step context
+
+    Returns:
+        Seldon Core deployment service
+    """
+    # Find a service created by a previous run of this step
+    step_env = cast(StepEnvironment, Environment()[STEP_ENVIRONMENT_NAME])
+    step_name = config.step_name or step_env.step_name
+
+    logger.info(
+        f"Loading last service deployed by step {step_name} and "
+        f"pipeline {step_env.pipeline_name}..."
+    )
+
+    try:
+        service = cast(
+            SeldonDeploymentService,
+            # TODO [HIGH]: catch errors that are raised because a previous step used
+            #   an integration that is no longer available
```

Comment:
Update_todo gh action only checks at 'src/zenml' and at 'tests' for these Todos. If this isn't doable this sprint, you should create the ticket in jira manually.

Reply:
I want to remake this step into a standard Seldon Core step in my next PR (it's part of the Seldon Core integration story). The TODOs will be reflected there as well, so I'll just leave them here for now, if that's ok with you.